pkgs/build-support/fetchpypilegacy/default.nix

```diff
@@ -2,8 +2,9 @@
 { runCommand
 , lib
 , python3
-}:
-{
+, cacert
+}@pkgs:
```

Comment:
I'd usually name this `args`, as it's not the nixpkgs attrset.

Reply:
My thinking is that this _is_ the package set, just a small subset of it.
I think `@args` would be more appropriate if it was further down the function stack.

Reply:
Suit yourself. The scope of the name is pretty small, so any argumentation over it from me is bikeshedding.